apis/Google.Datastore.V1Beta3/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -90,10 +91,11 @@ public void StructuredQuery()
             {
                 Filter = Filter.Equal("author", "Jane Austen")
             };
-            DatastoreQueryResults results = db.RunQuery(query);
-            // DatastoreQueryResults implements IEnumerable<Entity>, but you can
-            // call AsEntityResults(), AsBatches() or AsResponses() to see the query
-            // results in whatever way makes most sense for your application.
+            LazyDatastoreQuery results = db.RunQuery(query);
```

Comment:
The word "Lazy" in LazyDatastoreQuery helps communicate to readers the semantics of the query, but most of the time, the name of the type will not be visible.  9 times out of 10, I will write:
  var results = db.RunQuery(query);